config/jobs/kubernetes/sig-security/govulncheck-presubmit.yaml

```diff
@@ -0,0 +1,28 @@
+presubmits:
+  kubernetes/kubernetes:
+  - name: check-govulncheck-results
+    decorate: true
+    decoration_config:
+      timeout: 5m
+    path_alias: k8s.io/kubernetes
+    extra_refs:
+      - org: kubernetes
+        repo: sig-security
+        base_ref: main
+        workdir: true
+    always_run: false
+    optional: true
+    run_if_changed: '^(go.mod|go.sum|vendor)'
+    spec:
+      containers:
+      - image: golang
+        command:
+        - /bin/bash
+        args:
+        - -c
+        - |
```

Comment:
I think this pipe here as is will cause issues. The pipe in yaml indicates a multi-line value. However following this is the script value in a separate array element. See https://yaml.org/spec/1.2-old/spec.html#id2795688. 

I'd recommend either:

- Removing this line

or
-  remove the quotes and dash from the following line. i.e.
```
  args:
  - -c
  - |
    cd sig-security-tooling/govulncheck/hack/ && ./govulncheck-presubmit.sh
```

Reply:
Sure, thank you 👍